mailbox/grpc_noise_conn.go

```diff
@@ -215,6 +215,17 @@ func (c *NoiseGrpcConn) ClientHandshake(_ context.Context, _ string,
 		return nil, nil, err
 	}
 
+	// If the server has indicated that there will be authData sent in
+	// an Act Four, then read that now.
+	if c.noise.actFourSize > 0 {
+		actFour := make([]byte, c.noise.actFourSize)
```

Comment:
So re the comment above, we'd just read it as a normal encrypted message w/ the length prefix.